hs_core/tests/api/native/test_get_citation.py

```diff
@@ -129,3 +129,46 @@ def test_parse_citation_name(self):
         name = "Smith Tanner, John Morley"
         parsed_name = self.res.parse_citation_name(name)
         self.assertEqual(parsed_name, 'J. M. Smith Tanner, ')
+
+    def test_author_ordering(self):
```

Comment:
I was using this as a test code to make sure the citation does not change as a result of author order fix. We don't need this anymore.